### PR TITLE
[WIP] OSDOCS-7876: update greenboot directories

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-7876

Link to docs preview:
https://65163--docspreview.netlify.app/microshift/latest/microshift_running_apps/microshift-greenboot-workload-scripts

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: _unused_topics/microshift-updating-container-image-new-app.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
```
Comment (pmtk) [inline _unused_topics/microshift-updating-container-image-new-app.adoc:1]: Should this file be part of this PR?

File: modules/microshift-greenboot-create-health-check-script.adoc [hunk 2]
```diff
@@ -7,11 +7,11 @@
 [id="microshift-greenboot-app-health-check-script_{context}"]
 = How to create a health check script for your application
 
-You can create workload or application health check scripts in the text editor of your choice using the example in this documentation. Save the scripts in the `/etc/greenboot/check/required.d` directory. When a script in the `/etc/greenboot/check/required.d` directory exits with an error, greenboot triggers a reboot in an attempt to heal the system.
+You can create workload or application health check scripts in the text editor of your choice using the "Example workload health check script" as a template. Save the scripts in the `/usr/lib/greenboot/check/required.d` directory. When a script in the `/usr/lib/greenboot/check/required.d` directory exits with an error, greenboot triggers a reboot in an attempt to heal the system.
```
Comment (pmtk) [inline modules/microshift-greenboot-create-health-check-script.adoc:10]: Here's the thing. That path is read-only on the ostree systems. If someone wants their script to reside there, it should be packaged and shipped via RPM.

user can use their favorite editor to create a script in `/etc/greenboot` no problem (with sudo)

File: modules/microshift-greenboot-dir-structure.adoc [hunk 3]
```diff
@@ -24,18 +24,23 @@ Greenboot redirects script and health check output to the system log. When you a
 
 Returning a nonzero exit code from any script means that script has failed. Greenboot restarts the system a few times to retry the scripts before attempting to roll back to the previous version.
 
-* `/etc/greenboot/check/required.d` contains the health checks that must not fail.
+* `/usr/lib/greenboot/check/required.d` contains the health checks that must not fail.
```
Comment (pmtk) [inline modules/microshift-greenboot-dir-structure.adoc:27]: The old version is still true, failing script in both `/usr/lib/lib/greenboot/check/required.d` and `/etc/greenboot/check/required.d` can trigger reboot

File: modules/microshift-greenboot-dir-structure.adoc [hunk 4]
```diff
@@ -24,18 +24,23 @@ Greenboot redirects script and health check output to the system log. When you a
...
 
-** If the scripts fail, greenboot retries them three times by default. You can configure the number of retries in the `/etc/greenboot/greenboot.conf` file by setting the `GREENBOOT_MAX_BOOTS` parameter to the desired number of retries.
+** If the scripts fail, greenboot retries them three times by default. You can configure the number of retries in the `greenboot/greenboot.conf` file by setting the `GREENBOOT_MAX_BOOTS` parameter to the desired number of retries.
```
Comment (pmtk) [inline modules/microshift-greenboot-dir-structure.adoc:29]: We need to keep that `/etc/`

```suggestion
** If the scripts fail, greenboot retries them three times by default. You can configure the number of retries in the `/etc/greenboot/greenboot.conf` file by setting the `GREENBOOT_MAX_BOOTS` parameter to the desired number of retries.
```

File: modules/microshift-greenboot-microshift-health-script.adoc [hunk 5]
```diff
@@ -51,8 +51,8 @@ The `40_microshift_running_check.sh` health check script only performs validatio
 |`exit 1`
 |===
 
-[id="validation-wait-period"]
+[id="validation-wait-period_{context}"]
 == Validation wait period
 The wait period in each validation is five minutes by default. After the wait period, if the validation has not succeeded, it is declared a failure. This wait period is incrementally increased by the base wait period after each boot in the verification loop.
 
-* You can override the base-time wait period by setting the `MICROSHIFT_WAIT_TIMEOUT_SEC` environment variable in the `/etc/greenboot/greenboot.conf` configuration file. For example, you can change the wait time to three minutes by resetting the value to 180 seconds, such as `MICROSHIFT_WAIT_TIMEOUT_SEC=180`.
+* You can override the base-time wait period by setting the `MICROSHIFT_WAIT_TIMEOUT_SEC` environment variable in the `greenboot.conf` configuration file. For example, you can change the wait time to three minutes by resetting the value to 180 seconds, such as `MICROSHIFT_WAIT_TIMEOUT_SEC=180`.
```
Comment (pmtk) [inline modules/microshift-greenboot-microshift-health-script.adoc:58]:
```suggestion
* You can override the base-time wait period by setting the `MICROSHIFT_WAIT_TIMEOUT_SEC` environment variable in the `/etc/greenboot/greenboot.conf` configuration file. For example, you can change the wait time to three minutes by resetting the value to 180 seconds, such as `MICROSHIFT_WAIT_TIMEOUT_SEC=180`.
```

File: modules/microshift-greenboot-dir-structure.adoc [hunk 6]
```diff
@@ -6,13 +6,13 @@
 [id="microshift-greenboot-dir-structure_{context}"]
 = How greenboot uses directories to run scripts
 
-Health check scripts run from four `/etc/greenboot` directories. These scripts run in alphabetical order. Keep this in mind when you configure the scripts for your workloads.
+Health check scripts run from two directories. These scripts run in alphabetical order. Keep this in mind when you configure the scripts for your workloads.
```
Comment (pmtk) [inline modules/microshift-greenboot-dir-structure.adoc:9]: Well, we didn't know it at the time of writing previous doc, but four is actually correct:
- /usr/lib/greenboot/checks/required.d
- /usr/lib/greenboot/checks/wanted.d
- /etc/greenboot/checks/required.d
- /etc/greenboot/checks/wanted.d

File: _unused_topics/microshift-updating-container-image-new-app.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply _unused_topics/microshift-updating-container-image-new-app.adoc:1]: Nope. Removing.

**Comment by ShaunaDiaz** [comment]
@pmtk Revised, PTAL? Thank you!

**Comment by ShaunaDiaz** [comment]
Engineering issue closed as Obsolete. Doc issue closed also.

**Pull request closed without merging**